.env.example

```
@@ -8,5 +8,5 @@
 
 VITE_API_BASE_PATH=http://localhost:8080
 VITE_KEYCLOAK_URL=http://localhost:8180
-VITE_KEYCLOAK_REALM=spezi-study
-VITE_KEYCLOAK_CLIENT_ID=spezi-study-web
+VITE_KEYCLOAK_REALM=spezistudyplatform
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_KEYCLOAK_REALM key should go before the VITE_KEYCLOAK_URL key

(UnorderedKey)